.github/r_code/pdflatex_local_manual.r

```diff
@@ -0,0 +1,33 @@
+#' Comment out lines that prevent pdf from being generated locally and produce pdf
+#' @description Reads the tex file and comments out lines that prevent pdf from being
```

Comment:
There should be a full return after the title line and the description tag is not standard to include because it is known that the description starts on the third line unless the description spans multiple paragraphs.